addons/account/tests/test_account_all_l10n.py

```diff
@@ -28,8 +28,9 @@ def try_loading_patch(self, template_code, company, install_demo=True):
 
     # Install the requiriments
     l10n_mods = env['ir.module.module'].search([
-        ('name', '=like', 'l10n%'),
+        ('name', '=like', 'l10n_%'),
```

Comment:
Note that this is useless: `_` is a one-character wildcard in SQL

Reply:
This is actually crucial to avoid getting the 'l10n' module

You are right, this diff was remaining after some back-and-forth during testing and we decided to keep even if it wasn't intentional. Wasn't a really good idea since it caused some conflict when back-porting but it is merged anyway ;)